src/open_inwoner/scss/components/Appointments/Appointments.scss

```diff
@@ -0,0 +1,7 @@
+.appointments {
+  .card__body {
+    span.button:last-child {
+      bottom: var(--spacing-large) !important;
```

Comment:
We like to avoid "!important" - what if you try to make it more specific by adding the parents with  ">" selectors? Have not tested this but maybe this is specific enough: 
`.plugin-card > .card__body > .button.button--icon-before.button--transparent`